addons-l10n/en/editor-devtools.json

```diff
@@ -26,21 +23,19 @@
   "editor-devtools/ctrl-lr-desc": "Navigate to previous / next visited position in the script window (after using the navigate to block or find bar). This allows you to middle click a custom block to go to its definition, then press ctrl + Left to go back to where you were before.",
   "editor-devtools/costume-tab-features": "Costumes Tab Features",
   "editor-devtools/find-bar": "Find Bar",
-  "editor-devtools/find-bar-costume-desc": "Click to list all costumes by name, and type to locate one. Use the arrow keys or mouse to click a name to just straight to that costume.",
+  "editor-devtools/find-bar-costume-desc": "Click to list all costumes by name, and type to locate one. Use the arrow keys or mouse to click a name to jump straight to that costume.",
   "editor-devtools/ctrl-lr-costume-desc": "These keys navigate you to the previous / next costume in the sprite.",
   "editor-devtools/send-top-bottom": "Send to Top, Send to Bottom",
   "editor-devtools/send-top-bottom-desc": "Right click a costume and 2 new menu items are present. These can be used to send the clicked sprite to the top or bottom of the list of costumes for fast re-ordering.",
   "editor-devtools/youtube": "YouTube tutorials",
   "editor-devtools/orphaned": "{count, plural, one {Griffpatch: I found 1 orphaned reporter block. Shall I delete it for you?} other {Griffpatch: I found # orphaned reporter blocks. Shall I delete them for you?} }",
   "editor-devtools/unused-var": "{count, plural, one {Griffpatch: I found 1 unused local variable. Shall I delete it for you? Here it is:\n} other {Griffpatch: I found # unused local variables. Shall I delete them for you? Here they are:\n} }",
```

Comment:
Now that we're editing this file, can we replace all the instances of `Griffpatch` with `Developer tools`?

Reply:
for consistency with disable-auto-save, it should be `Message by \"developer tools\" addon from Scratch Addons:`

Reply:
@RedGuy12 No, keep in mind the devtools extension (independent from SA, kinda) uses these same strings